src/core/storage/providers/local/local_file_entity.ts

```diff
@@ -31,7 +33,7 @@ export class LocalFileEntity implements FileEntity{
     async upload(body: NodeJS.ReadableStream): Promise<FileEntity> {
         
         // Get the directory from file path
-        const fullUploadPath = this.rootUploadPath+this.filePath;
+        const fullUploadPath = this.serverRoot+this.rootUploadPath+this.filePath;
         const directoryPath = fullUploadPath.substring(0,fullUploadPath.indexOf(this.fileName)-1);
```

Comment:
path.dirname(fullUploadPath) will give you the directory.

Reply:
this is fixed